lms/templates/courseware/tab_default.html

```diff
@@ -0,0 +1,19 @@
+## mako
+<%namespace name='static' file='/static_content.html'/>
+<%!
+from django.utils.translation import ugettext as _
+from django.core.urlresolvers import reverse
+%>
+<li>
+    <a href="${tab.link_func(course, reverse) | h}" class="${tab_class}">
+    ${_(tab.name) | h}
+    % if tab_is_active:
+        <span class="sr">, current location</span>
```

Comment:
@cptvitamin can you comment on the need for i18n for screen reader span elements?